src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,14 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        if (value == 0) {
+            return "0";
+        }
+        StringBuilder isBinary = new StringBuilder();
```

Comment:
`is...` usually, we use with boolean variables. Let's rename variable to `binaryString` or `numericalString` or something else

Reply:
ok, done